blimp-live-widgets/src/components/Weather/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './styles.css'
+
+export default class Weather extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      weather: {
+        city: "Waterloo",
+        temperature: 0,
+        main: "None",
+        description: "None",
+      },
+      isLoaded: false
+    };
+  }
+
+   componentDidMount() {
+    fetch("http://api.openweathermap.org/data/2.5/weather?q=Waterloo,CA&appid=d79a0d378573a8b844d4a8b7e1c963d6")
+      .then(function(res){
+        return res.json();
+      })
+      .then(
+        (result) => {
+          console.log("TEST")
+
+          console.log(result)
+          this.setState({
+            weather: {
+              city: result.name,
+              temperature: Math.round(result.main.temp - 273),
```

Comment:
https://openweathermap.org/current

"Temperature is available in Fahrenheit, Celsius and Kelvin units." - I think we can get it to request in diff units. Maybe we can pass the type in dynamically and then later extend this to allow user input.